.github/workflows/publish-version.yml

```diff
@@ -159,23 +97,14 @@ jobs:
         run: |
           set -x
 
-          version=${{ steps.outputs.outputs.version }}
-          currentVersion=${{ steps.outputs.outputs.current-version }}
-
-          # Bump the crate version
-          sed -i "s#^version = \".*\"#version = \"${version}\"#" Cargo.toml
-          sed -i "s#^version = \".*\"#version = \"${version}\"#" crates/sdk/Cargo.toml
-          sed -i "s#^version = \".*\"#version = \"${version}\"#" crates/core/Cargo.toml
-
-          # Update dependency versions
-          sed -i "s#surrealdb = { version = \"=${currentVersion}\"#surrealdb = { version = \"=${version}\"#" Cargo.toml
-          sed -i "s#surrealdb-core = { version = \"=${currentVersion}\"#surrealdb-core = { version = \"=${version}\"#" Cargo.toml
-
-          # Update Cargo.lock without updating dependency versions
-          cargo check --no-default-features --features storage-mem
+          ENVIRONMENT="not-release" \
```

Comment:
I think it might be safer to default to `nightly` here instead of `not-release`. In case this ends up not getting replaced, I think it's better to do a nightly deployment.

Reply:
That's a good idea, this call only updates version information, but just in case something gets modified incorrectly in the script in the future, we don't want stray published crates.